src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -34,6 +34,14 @@
 
 {%- block form_widget_simple -%}
     {%- set type = type|default('text') -%}
+  {% if attr.counter is defined %}
```

Comment:
You may use a macro for it: http://twig.sensiolabs.org/doc/tags/macro.html#macro